sandbox/libs/authn/src/main/java/org/opensearch/authn/jwt/JwtVendorTestUtils.java

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
Can this class be moved into `src/test/java`?

Reply:
No problem